packages/test/src/test-workflow-unhandled-rejection-crash.ts

```diff
@@ -17,13 +17,17 @@ if (RUN_INTEGRATION_TESTS) {
       taskQueue,
       args: [{ crashWorker: true }],
     });
-    await t.throwsAsync(worker.run(), {
-      instanceOf: UnexpectedError,
-      message:
-        'Worker thread shut down prematurely, this could be caused by an' +
-        ' unhandled rejection in workflow code that could not be' +
-        ' associated with a workflow run',
-    });
-    await handle.terminate();
+    try {
+      await t.throwsAsync(worker.run(), {
+        instanceOf: UnexpectedError,
+        message:
+          'Workflow Worker Thread exited prematurely: UnhandledRejectionError: ' +
```

Comment:
I would say that's quite intentional and reads clearly to me, but if you have suggestions, go ahead.

For context: "Worker Thread" is the name of the Node.js' threading API, so just saying "Worker Thread exited prematurely" would be overly generic. Also, only Workflow Tasks get run in Worker Threads, because that's part of the sandboxing mechanism. So being specific on this specific term feels appropriate to me. I want this to be understood as "a thread used by your Temporal Worker for running Workflows".